app/src/play/java/de/danoeh/antennapod/preferences/PreferenceControllerFlavorHelper.java

```diff
@@ -26,7 +32,16 @@ public static void setupFlavoredUI(PlaybackPreferencesFragment ui) {
                     return false;
                 }
             }
+            displayRestartRequiredDialog(ui.requireContext());
```

Comment:
Is this line actually needed? The dialog is displayed even when disabling cast support, what is handled fine without a restart.

Reply:
It's taken out - Disabling cast support does not require a restart strictly speaking. 

I was just err on being safe, as the restart will ensure all cast-related code wasn't run at all in the process.